File: Android project/InstagramCloneJava2/app/src/main/java/com/mirkamol/instagramclonejava/utils/Logger.java
package com.mirkamol.instagramclonejava.utils;

import android.util.Log;

public class Logger {
    Boolean IS_TESTER = true;
   public void d (String tag, String msg){
        if (IS_TESTER){
            Log.d(tag, msg);
        }
    }

    public void i(String tag, String msg){
        if (IS_TESTER){
            Log.d(tag, msg);
        }
    }

  public   void v (String tag, String msg){
        if (IS_TESTER){
            Log.d(tag, msg);
        }
    }

   public void e(String tag, String msg){
        if (IS_TESTER){
            Log.d(tag, msg);
        }
    }

}

File: Android project/Lesson6VolleyNetWorking/app/src/main/java/com/mirkamol/lesson6volleynetworking/utils/Logger.kt
package com.mirkamol.lesson6volleynetworking.utils

import android.util.Log
import com.mirkamol.lesson6volleynetworking.network.VolleyHttp

class Logger {
    companion object {

        fun d(tag: String, msg: String) {
            if (VolleyHttp.IS_TESTER) Log.d(tag, msg)
        }

        fun i(tag: String, msg: String) {
            if (VolleyHttp.IS_TESTER) Log.i(tag, msg)
        }

        fun v(tag: String, msg: String) {
            if (VolleyHttp.IS_TESTER) Log.v(tag, msg)
        }

        fun e(tag: String, msg: String) {
            if (VolleyHttp.IS_TESTER) Log.e(tag, msg)
        }
    }
}